Gemfile

```diff
@@ -56,6 +55,9 @@ gem 'rack-cors', '~> 1.0.2', require: 'rack/cors'
 gem 'jwt', '~> 2.2.1'
 gem 'graphql', '~> 1.8.0'
 gem 'graphql-batch'
+# this one is a dependecy for x-editable-rails
+# when removing turbolinks, coffee-script is removed as well
+gem 'coffee-rails', '~> 5.0.0' 
```

Comment:
Is this a build time dependency or is it needed for all installs? I suspect coffee-script is only needed to build assets so it would belong in ` bundler.d/assets.rb`

Reply:
makes sense :+1: